packages/medusa/yarn.lock

```diff
@@ -3,1064 +3,886 @@
 
 
 "@babel/cli@^7.14.3":
```

Comment:
question: Why is there those version upgrade here?

Reply:
reverted to develop version